src/components/Viewer.tsx

```diff
@@ -69,10 +83,40 @@ export default function Viewer({ handleTimeChange }: ViewerProps): ReactNode {
         }
     }, [page, setViewportToContainerSize, heightResized, trajectoryName]);
 
+    useEffect(() => {
+        const el = container.current;
+        el?.addEventListener("mousedown", (event) => {
+            if (!userHasInteracted) {
+                setFirstMousePosition({
+                    x: event.clientX,
+                    y: event.clientY,
+                });
+            }
+        });
+        container.current?.addEventListener("mouseup", (event) => {
+            if (
+                firstMousePosition.x !== event.clientX &&
+                firstMousePosition.y !== event.clientY
+            ) {
+                setUserHasInteracted(true);
+            }
+        });
```

Comment:
Should these be wrapped in an, `if (trajectoryName === LIVE_SIMULATION_NAME)` check? I forget if `Viewer` is also the component handling the 2D simulations, but if it is, clicking and dragging on the viewer if it's not viewing the live simulation would also disable the popup later.

Reply:
yeah, that's a good point